src/PrestaShopBundle/Resources/views/Admin/Sell/Catalog/Product/Combination/paginated_list.html.twig

```diff
@@ -35,11 +35,19 @@
     ></div>
 
     <div id="combination-edit-modal"
+         data-empty-image="{{ asset('themes/new-theme/img/empty_state/combinations_history.png') }}"
          data-translations="{{ {
-           'modal.save': 'Save'|trans({}, 'Admin.Actions'),
-           'modal.close': 'Close'|trans({}, 'Admin.Actions'),
-           'modal.previous': 'Previous combination'|trans({}, 'Admin.Catalog.Feature'),
-           'modal.next': 'Next combination'|trans({}, 'Admin.Catalog.Feature'),
+          'modal.save': 'Save'|trans({}, 'Admin.Actions'),
+          'modal.close': 'Close'|trans({}, 'Admin.Actions'),
+          'modal.previous': 'Previous combination'|trans({}, 'Admin.Catalog.Feature'),
+          'modal.next': 'Next combination'|trans({}, 'Admin.Catalog.Feature'),
+          'modal.history.editedCombination': 'Edited combinations (%editedNb%)'|trans({}, 'Admin.Catalog.Feature'),
+          'modal.history.confirmTitle': 'Discard changes?'|trans({}, 'Admin.Catalog.Notification'),
+          'modal.history.confirmBody': 'All unsaved modifications made on the combination [1]%combinationName%[/1] will be lost.'|trans({'[1]': '<span class="font-weight-bold">', '[/1]': '</span>'}, 'Admin.Catalog.Notification'),
+          'modal.cancel': 'Cancel'|trans({}, 'Admin.Actions'),
+          'modal.confirm': 'Discard'|trans({}, 'Admin.Actions'),
+          'modal.history.empty': 'Here, you can have a look at the combinations you have edited. The list will reset when you exit the editing window.'|trans({}, 'Admin.Catalog.Feature'),
```

Comment:
```suggestion
          'modal.history.empty': 'Here, you can have a look at the combinations you have edited. The list will reset when you close the edition popin.'|trans({}, 'Admin.Catalog.Feature'),
```
Just a suggestion for @Julievrz But maybe it's too technical?

Reply:
@jolelievre I think it sounds a bit too technical yes but we can replace "exit" with "close" which is even easier to understand. :)

Reply:
```suggestion
          'modal.history.empty': 'Here, you can have a look at the combinations you have edited. The list will reset when you close the editing window.'|trans({}, 'Admin.Catalog.Feature'),
```